src/Aspire.Dashboard/Components/Pages/Resources.razor

```diff
@@ -87,6 +87,7 @@
                                         Loading="_isLoading"
                                         ShowHover="true"
                                         TGridItem="ResourceViewModel"
+                                        ItemKey="@(r => r.Name)"
```

Comment:
Did you test this with replicas?

Reply:
Yes.

Have replicas been tested with a Waiting status - do they have a unique name by that point?

Reply:
I think we throw